lxd/config/generate/lxddoc.go

```diff
@@ -191,7 +192,38 @@ func parse(path string, outputYAMLPath string, excludedPaths []string) (*doc, er
 		return nil, err
 	}
 
-	yamlDoc.Configs = projectEntries
+	// Within each group, alphabetically sort the entries by key (config option key)
+	orderedProjectEntries := make(map[string][]any, len(projectEntries))
+	for groupKey, entries := range projectEntries {
+		var sortedConfigOptionKeysPerGroup []string
+		for _, configEntry := range entries {
+			c := configEntry.(map[string]any)
+			for k := range c {
```

Comment:
```suggestion
			for k := range configEntry.(map[string]any) {
```
There's no need for the `c` variable.